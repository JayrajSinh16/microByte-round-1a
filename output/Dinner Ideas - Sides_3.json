{
  "title": "o 1/4 cup olive oil  Instructions :",
  "outline": [
    {
      "level": "H1",
      "text": " Ingredients :",
      "page": 1
    },
    {
      "level": "H2",
      "text": " Ingredients :",
      "page": 1
    },
    {
      "level": "H1",
      "text": "Hummus",
      "page": 2
    },
    {
      "level": "H2",
      "text": " Ingredients :",
      "page": 2
    },
    {
      "level": "H3",
      "text": " Ingredients :",
      "page": 2
    },
    {
      "level": "H3",
      "text": " Ingredients :",
      "page": 2
    },
    {
      "level": "H3",
      "text": " Ingredients :",
      "page": 3
    },
    {
      "level": "H3",
      "text": " Ingredients :",
      "page": 3
    },
    {
      "level": "H1",
      "text": "Kachumbari",
      "page": 4
    },
    {
      "level": "H2",
      "text": " Ingredients :",
      "page": 4
    },
    {
      "level": "H3",
      "text": " Ingredients :",
      "page": 4
    },
    {
      "level": "H3",
      "text": " Ingredients :",
      "page": 4
    },
    {
      "level": "H3",
      "text": " Instructions :",
      "page": 5
    },
    {
      "level": "H3",
      "text": " Ingredients :",
      "page": 5
    },
    {
      "level": "H3",
      "text": " Ingredients :",
      "page": 5
    },
    {
      "level": "H1",
      "text": "Kimchi",
      "page": 6
    },
    {
      "level": "H2",
      "text": " Ingredients :",
      "page": 6
    },
    {
      "level": "H3",
      "text": " Ingredients :",
      "page": 6
    },
    {
      "level": "H3",
      "text": " Ingredients :",
      "page": 6
    },
    {
      "level": "H1",
      "text": "Larb",
      "page": 7
    },
    {
      "level": "H2",
      "text": " Ingredients :",
      "page": 7
    },
    {
      "level": "H3",
      "text": " Ingredients :",
      "page": 7
    },
    {
      "level": "H3",
      "text": " Ingredients :",
      "page": 7
    },
    {
      "level": "H3",
      "text": " Ingredients :",
      "page": 8
    },
    {
      "level": "H3",
      "text": " Ingredients :",
      "page": 8
    },
    {
      "level": "H1",
      "text": "Mandazi",
      "page": 9
    },
    {
      "level": "H2",
      "text": " Ingredients :",
      "page": 9
    },
    {
      "level": "H3",
      "text": " Ingredients :",
      "page": 9
    },
    {
      "level": "H3",
      "text": " Ingredients :",
      "page": 9
    },
    {
      "level": "H3",
      "text": " Ingredients :",
      "page": 10
    },
    {
      "level": "H3",
      "text": " Ingredients :",
      "page": 10
    },
    {
      "level": "H3",
      "text": " Ingredients :",
      "page": 10
    },
    {
      "level": "H3",
      "text": " Ingredients :",
      "page": 11
    },
    {
      "level": "H3",
      "text": " Ingredients :",
      "page": 11
    },
    {
      "level": "H1",
      "text": "Msemen",
      "page": 12
    },
    {
      "level": "H2",
      "text": " Ingredients :",
      "page": 12
    },
    {
      "level": "H3",
      "text": " Ingredients :",
      "page": 12
    },
    {
      "level": "H3",
      "text": " Ingredients :",
      "page": 12
    },
    {
      "level": "H3",
      "text": " Ingredients :",
      "page": 13
    },
    {
      "level": "H3",
      "text": " Ingredients :",
      "page": 13
    },
    {
      "level": "H3",
      "text": " Ingredients :",
      "page": 13
    },
    {
      "level": "H3",
      "text": " Ingredients :",
      "page": 14
    },
    {
      "level": "H3",
      "text": " Ingredients :",
      "page": 14
    },
    {
      "level": "H3",
      "text": " Ingredients :",
      "page": 14
    },
    {
      "level": "H3",
      "text": " Ingredients :",
      "page": 15
    },
    {
      "level": "H3",
      "text": " Ingredients :",
      "page": 15
    }
  ]
}