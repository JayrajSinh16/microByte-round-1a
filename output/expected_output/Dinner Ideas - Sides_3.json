{
    "title": "Dinner - Sides",
    "outline": [
        {
            "level": "H1",
            "text": "Horta",
            "page": 1
        },
        {
            "level": "H2",
            "text": "Instructions",
            "page": 1
        },
        {
            "level": "H2",
            "text": "Ingredients",
            "page": 1
        },
        {
            "level": "H1",
            "text": "Hot and Sour Soup",
            "page": 1
        },
        {
            "level": "H2",
            "text": "Ingredients",
            "page": 1
        },
        {
            "level": "H2",
            "text": "Instructions",
            "page": 1
        },
        {
            "level": "H1",
            "text": "Hummus",
            "page": 2
        },
        {
            "level": "H2",
            "text": "Ingredients",
            "page": 2
        },
        {
            "level": "H2",
            "text": "Instructions",
            "page": 2
        },
        {
            "level": "H1",
            "text": "Japchae",
            "page": 2
        },
        {
            "level": "H2",
            "text": "Ingredients",
            "page": 2
        },
        {
            "level": "H2",
            "text": "Instructions",
            "page": 2
        },
        {
            "level": "H1",
            "text": "Jerk Vegetables",
            "page": 2
        },
        {
            "level": "H2",
            "text": "Ingredients",
            "page": 2
        },
        {
            "level": "H2",
            "text": "Instructions",
            "page": 3
        },
        {
            "level": "H1",
            "text": "Johnny Cakes",
            "page": 3
        },
        {
            "level": "H2",
            "text": "Ingredients",
            "page": 3
        },
        {
            "level": "H2",
            "text": "Instructions",
            "page": 3
        },
        {
            "level": "H1",
            "text": "Jollof Rice",
            "page": 3
        },
        {
            "level": "H2",
            "text": "Ingredients",
            "page": 3
        },
        {
            "level": "H2",
            "text": "Instructions",
            "page": 3
        },
        {
            "level": "H1",
            "text": "Kachumbari",
            "page": 4
        },
        {
            "level": "H2",
            "text": "Ingredients",
            "page": 4
        },
        {
            "level": "H2",
            "text": "Instructions",
            "page": 4
        },
        {
            "level": "H1",
            "text": "Käsespätzle (Cheese Noodles)",
            "page": 4
        },
        {
            "level": "H2",
            "text": "Ingredients",
            "page": 4
        },
        {
            "level": "H2",
            "text": "Instructions",
            "page": 4
        },
        {
            "level": "H1",
            "text": "Kartoffelsalat (Potato Salad)",
            "page": 4
        },
        {
            "level": "H2",
            "text": "Ingredients",
            "page": 4
        },
        {
            "level": "H2",
            "text": "Instructions",
            "page": 5
        },
        {
            "level": "H1",
            "text": "Keftedes",
            "page": 5
        },
        {
            "level": "H2",
            "text": "Ingredients",
            "page": 5
        },
        {
            "level": "H2",
            "text": "Instructions",
            "page": 5
        },
        {
            "level": "H1",
            "text": "Kibbeh",
            "page": 5
        },
        {
            "level": "H2",
            "text": "Ingredients",
            "page": 5
        },
        {
            "level": "H2",
            "text": "Instructions",
            "page": 5
        },
        {
            "level": "H1",
            "text": "Kimchi",
            "page": 6
        },
        {
            "level": "H2",
            "text": "Ingredients",
            "page": 6
        },
        {
            "level": "H2",
            "text": "Instructions",
            "page": 6
        },
        {
            "level": "H1",
            "text": "Kongnamul Muchim",
            "page": 6
        },
        {
            "level": "H2",
            "text": "Ingredients",
            "page": 6
        },
        {
            "level": "H2",
            "text": "Instructions",
            "page": 6
        },
        {
            "level": "H1",
            "text": "Labneh",
            "page": 6
        },
        {
            "level": "H2",
            "text": "Ingredients",
            "page": 6
        },
        {
            "level": "H2",
            "text": "Instructions",
            "page": 6
        },
        {
            "level": "H1",
            "text": "Larb",
            "page": 7
        },
        {
            "level": "H2",
            "text": "Ingredients",
            "page": 7
        },
        {
            "level": "H2",
            "text": "Instructions",
            "page": 7
        },
        {
            "level": "H1",
            "text": "Lo Mein",
            "page": 7
        },
        {
            "level": "H2",
            "text": "Ingredients",
            "page": 7
        },
        {
            "level": "H2",
            "text": "Instructions",
            "page": 7
        },
        {
            "level": "H1",
            "text": "Loubia",
            "page": 7
        },
        {
            "level": "H2",
            "text": "Ingredients",
            "page": 7
        },
        {
            "level": "H2",
            "text": "Instructions",
            "page": 8
        },
        {
            "level": "H1",
            "text": "Macaroni and Cheese",
            "page": 8
        },
        {
            "level": "H2",
            "text": "Ingredients",
            "page": 8
        },
        {
            "level": "H2",
            "text": "Instructions",
            "page": 8
        },
        {
            "level": "H1",
            "text": "Manakish",
            "page": 8
        },
        {
            "level": "H2",
            "text": "Ingredients",
            "page": 8
        },
        {
            "level": "H2",
            "text": "Instructions",
            "page": 8
        },
        {
            "level": "H1",
            "text": "Mandazi",
            "page": 9
        },
        {
            "level": "H2",
            "text": "Ingredients",
            "page": 9
        },
        {
            "level": "H2",
            "text": "Instructions",
            "page": 9
        },
        {
            "level": "H1",
            "text": "Mandu",
            "page": 9
        },
        {
            "level": "H2",
            "text": "Ingredients",
            "page": 9
        },
        {
            "level": "H2",
            "text": "Instructions",
            "page": 9
        },
        {
            "level": "H1",
            "text": "Mango Sticky Rice",
            "page": 9
        },
        {
            "level": "H2",
            "text": "Ingredients",
            "page": 9
        },
        {
            "level": "H2",
            "text": "Instructions",
            "page": 9
        },
        {
            "level": "H1",
            "text": "Mapo Tofu",
            "page": 10
        },
        {
            "level": "H2",
            "text": "Ingredients",
            "page": 10
        },
        {
            "level": "H2",
            "text": "Instructions",
            "page": 10
        },
        {
            "level": "H1",
            "text": "Mashed Potatoes",
            "page": 10
        },
        {
            "level": "H2",
            "text": "Ingredients",
            "page": 10
        },
        {
            "level": "H2",
            "text": "Instructions",
            "page": 10
        },
        {
            "level": "H1",
            "text": "Minestrone",
            "page": 10
        },
        {
            "level": "H2",
            "text": "Ingredients",
            "page": 10
        },
        {
            "level": "H2",
            "text": "Instructions",
            "page": 11
        },
        {
            "level": "H1",
            "text": "Miso Soup",
            "page": 11
        },
        {
            "level": "H2",
            "text": "Ingredients",
            "page": 11
        },
        {
            "level": "H2",
            "text": "Instructions",
            "page": 11
        },
        {
            "level": "H1",
            "text": "Moqueca de Banana",
            "page": 11
        },
        {
            "level": "H2",
            "text": "Ingredients",
            "page": 11
        },
        {
            "level": "H2",
            "text": "Instructions",
            "page": 11
        },
        {
            "level": "H1",
            "text": "Msemen",
            "page": 12
        },
        {
            "level": "H2",
            "text": "Ingredients",
            "page": 12
        },
        {
            "level": "H2",
            "text": "Instructions",
            "page": 12
        },
        {
            "level": "H1",
            "text": "Muhammara",
            "page": 12
        },
        {
            "level": "H2",
            "text": "Ingredients",
            "page": 12
        },
        {
            "level": "H2",
            "text": "Instructions",
            "page": 12
        },
        {
            "level": "H1",
            "text": "Mutabal",
            "page": 12
        },
        {
            "level": "H2",
            "text": "Ingredients",
            "page": 12
        },
        {
            "level": "H2",
            "text": "Instructions",
            "page": 13
        },
        {
            "level": "H1",
            "text": "Naan",
            "page": 13
        },
        {
            "level": "H2",
            "text": "Ingredients",
            "page": 13
        },
        {
            "level": "H2",
            "text": "Instructions",
            "page": 13
        },
        {
            "level": "H1",
            "text": "Nom Du Du (Green Papaya Salad)",
            "page": 13
        },
        {
            "level": "H2",
            "text": "Ingredients",
            "page": 13
        },
        {
            "level": "H2",
            "text": "Instructions",
            "page": 13
        },
        {
            "level": "H1",
            "text": "Nuoc Cham",
            "page": 13
        },
        {
            "level": "H2",
            "text": "Ingredients",
            "page": 13
        },
        {
            "level": "H2",
            "text": "Instructions",
            "page": 14
        },
        {
            "level": "H1",
            "text": "Oi Muchim",
            "page": 14
        },
        {
            "level": "H2",
            "text": "Ingredients",
            "page": 14
        },
        {
            "level": "H2",
            "text": "Instructions",
            "page": 14
        },
        {
            "level": "H1",
            "text": "Onigiri (Rice Balls)",
            "page": 14
        },
        {
            "level": "H2",
            "text": "Ingredients",
            "page": 14
        },
        {
            "level": "H2",
            "text": "Instructions",
            "page": 14
        },
        {
            "level": "H1",
            "text": "Pão de Queijo",
            "page": 14
        },
        {
            "level": "H2",
            "text": "Ingredients",
            "page": 14
        },
        {
            "level": "H2",
            "text": "Instructions",
            "page": 15
        },
        {
            "level": "H1",
            "text": "Pad See Ew",
            "page": 15
        },
        {
            "level": "H2",
            "text": "Ingredients",
            "page": 15
        },
        {
            "level": "H2",
            "text": "Instructions",
            "page": 15
        },
        {
            "level": "H1",
            "text": "Pad Thai",
            "page": 15
        },
        {
            "level": "H2",
            "text": "Ingredients",
            "page": 15
        },
        {
            "level": "H2",
            "text": "Instructions",
            "page": 15
        }
    ]
}